receiver/otelarrowreceiver/README.md

```diff
@@ -77,6 +77,16 @@ Several common configuration structures provide additional capabilities automati
 - [gRPC settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configgrpc/README.md)
 - [TLS and mTLS settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md)
 
+### Admission Control Configuration
+
+In the `admission` configuration block the following settings are available:
+
+- `request_limit_mib` (default: 128): limits the number of requests that are received by the stream based on request size information available. This should not be confused with `arrow.memory_limit_mib` which limits allocations made by the consumer when translating arrow records into pdata objects. i.e. request size is used to control how much traffic we admit, but does not control how much memory is used during request processing.
```

Comment:
Nit: It would be helpful to clarify whether this limit represents the size of the compressed request or the uncompressed request.